backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -599,6 +666,23 @@ public Task<Entry> UpdateEntry(Guid id, UpdateObjectInput<Entry> update)
         return Task.FromResult(FromLexEntry(lexEntry));
     }
 
+
+
+    public async Task<Entry> UpdateEntry(Entry entry)
+    {
+        ValidateVersion(entry);
+        InvalidateVersion(entry);
```

Comment:
this is here because we don't have actual version tracking in FW. So rather than potentially overwriting changes made by another caller of the API we will throw if the version we are trying to update with is out of date.